docs/import/add_disaster.js

```diff
@@ -0,0 +1,253 @@
+import {eeLegacyPathPrefix, gdEePathPrefix} from '../ee_paths.js';
+import {getFirestoreRoot} from '../firestore_document.js';
+
+export {enableWhenReady, SENTINEL_NEW_DISASTER_VALUE, toggleState};
+// Visible for testing
+export {
+  addDisaster,
+  createAssetPickers,
+  createOptionFrom,
+  disasters,
+  emptyCallback,
+  getAssetsFromEe,
+  stateAssets,
+  writeNewDisaster,
+};
+
+// Currently a map of disaster name to states. This pulls once on firebase
+// authentication and then makes local updates afterwards so we don't need to
+// wait on firebase writes to read new info.
+const disasters = new Map();
+
+// Map of state to list of known assets
+const stateAssets = new Map();
+
+const SENTINEL_OPTION_VALUE = '...';
+const SENTINEL_NEW_DISASTER_VALUE = 'NEW DISASTER';
+
+/**
+ * Populates the disaster picker with disasters from firestore, and enables
+ * the ability to add a new disaster.
+ * @return {Promise<firebase.firestore.QuerySnapshot>}
+ */
+function enableWhenReady() {
+  // enable (currently hidden) add disaster button now that firestore is ready.
+  const addDisasterButton = $('#add-disaster-button');
+  addDisasterButton.prop('disabled', false);
+  addDisasterButton.on('click', addDisaster);
+
+  // populate disaster picker.
+  return getFirestoreRoot()
+      .collection('disaster-metadata')
+      .get()
+      .then((querySnapshot) => {
+        const disasterPicker = $('#disaster');
+        disasterPicker.append(createOptionFrom(SENTINEL_OPTION_VALUE));
+        querySnapshot.forEach((doc) => {
```

Comment:
Can we sort this in reverse order so most recent is at the top? Then we can also leave off the placeholder, which should simplify the code

Reply:
Done. Most recent is automatically displayed when you open the page now.